components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -15,10 +15,39 @@
 ~ specific language governing permissions and limitations
 ~ under the License.
 -->
+<script src="codemirror/lib/codemirror.js"></script>
+<script src="codemirror/keymap/sublime.js"></script>
+<link rel="stylesheet" href="codemirror/lib/codemirror.css">
+<script src="codemirror/mode/javascript/javascript.js"></script>
+<script src="codemirror/addon/hint/show-hint.js"></script>
+<script src="codemirror/addon/hint/javascript-hint.js"></script>
+<link rel="stylesheet" href="codemirror/addon/hint/show-hint.css">
+<script src="codemirror/addon/hint/anyword-hint.js"></script>
 <script>
-	function clearText() {
-        document.getElementById("scriptTextarea").value = ""; // clear scriptTextarea
-	}
+    jQuery(document).ready(function () {
+        var myCodeMirror = CodeMirror.fromTextArea(scriptTextarea, { //Adding CodeMirror IDE
+            lineNumbers: true,
+            mode: "javascript",
+            keymap: "sublime",
+            lineWiseCopyCut: true,
+            pasteLinesPerSelection: true,
+            extraKeys: {"Ctrl-Space": "autocomplete"},
+            indentWithTabs: true
+        });
+    });
+
+    editor.on('cursorActivity', function () {							// Giving custom hint list
+        var options = {
+            hint: function () {
+                return {
+                    from: editor.getDoc().getCursor(),
+                    to: editor.getDoc().getCursor(),
+                    list: ['acr', 'ACR selected']
```

Comment:
Our current inbuilt function is "selectAcrFrom" not "acr", so please change it accordingly.